.github/workflows/ci.yml

```
@@ -50,4 +50,4 @@ jobs:
         run: pytest --cov=removestar . -vv
 
       - name: Upload coverage report
-        uses: codecov/codecov-action@v3.1.4
+        uses: codecov/codecov-action@v5.1.1
```

Please revert this line so the rest of these upgrades can be merged.
```suggestion
        uses: codecov/codecov-action@v3.1.4
```
The change on this line is blocked by https://github.com/asmeurer/removestar/pull/64#discussion_r1856680372